library/classes/Contact.class.php

```diff
@@ -0,0 +1,95 @@
+<?php
```

Comment:
would store new classes somewhere in src tree with name of file with class name only like Contact.php
(long term goal is all classes migrate to psr4 compatible in src tree including the legacy ones based on ORDataObject)